test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Order.html

```diff
@@ -70,7 +70,7 @@
         </div>
         <div>
             <label class="order" for="Customer_Gender">Gender</label>
-            <input type="radio" value="Male" data-val="true" data-val-required="The Model field is required." id="Customer_Gender" name="Customer.Gender" /> Male
+            <input type="radio" value="Male" data-val="true" data-val-required="The Gender field is required." id="Customer_Gender" name="Customer.Gender" /> Male
```

Comment:
That's right -- we already had a functional test for this scenario. It was just looking for a broken error message.